circle.yml

```diff
@@ -1653,11 +1653,11 @@ linux-workflow: &linux-workflow
           - build-binary
           - build-npm-package
     - test-binary-against-recipe-pull-request:
-        name: Test fetch polyfill
+        name: Test cypress run parsing
         filters:
           branches:
             only:
-              - polyfill-fetch
+              - cli-to-module-api-7760
```

Comment:
don't forget to remove this

Reply:
nah, I will leave it in. It won't be running on `develop` or other branches, but will be available when needed.

Reply:
similar to the other one, plz add a comment for how someone would use this so it doesn't get accidentally deleted or ignored